lib/generators/active_fedora/noid/install_generator.rb

```diff
@@ -0,0 +1,36 @@
+module ActiveFedora
+  module Noid
+    class InstallGenerator < Rails::Generators::Base
+      include Rails::Generators::Migration
+      source_root File.expand_path('../templates', __FILE__)
+
+      desc <<-END_OF_DESC
+Copies DB migrations
+      END_OF_DESC
+
+      # Implement the required interface for Rails::Generators::Migration.
+      # taken from http://github.com/rails/rails/blob/master/activerecord/lib/generators/active_record.rb
+      def self.next_migration_number(path)
+        if @prev_migration_nr
+          @prev_migration_nr += 1
+        else
+          last_migration = Dir[File.join(path, '*.rb')].sort.last
+          @prev_migration_nr = if last_migration
+                                 last_migration.sub(File.join(path, '/'), '').to_i + 1
+                               else
+                                 Time.zone.now.utc.strftime("%Y%m%d%H%M%S").to_i
+                               end
+        end
+        @prev_migration_nr.to_s
+      end
+
+      def banner
+        say_status('info', 'Installing ActiveFedora::Noid', :blue)
+      end
+
+      def migrations
```

Comment:
Not sure what you mean.   I do not see any such task.  There is no `active_fedora-noid` task namespace.  In the context of the `.internal_test_app`, there is `active_fedora:noid:install` but not `active_fedora:noid:install:migrations`:
```
$ rails g active_fedora:noid:install:migrations
Could not find generator 'active_fedora:noid:install:migrations'. Maybe you meant 'active_fedora:noid:install', 'active_record:migration' or 'active_fedora:config'
Run `rails generate --help` for more options.
```

Are you thinking of something else?  Again, this is basically following the pattern laid out in Sufia.